modules/ROOT/pages/ch2-api.adoc

```diff
@@ -0,0 +1,376 @@
+= CloudHub 2.0 REST API
+
+*Last update* 02-Mar-22
+
+Added all the endpoints from https://github.com/mulesoft/rtf-management-service/blob/master/assets/raml/public/api.raml[GitHub].
+
+* *Private Spaces*
+** Get the list of private spaces in the organization.
+** Create a private space.
+** Delete the private space.
+** Get the private space. *What does this mean?*
+** Update an existing private space.
+
+* *Connections*
+** Get the list of connections inside the private space.
+** Create a private space connection. *What is this?*
```

Comment:
A "Connection" contains 1 or more VPNs. A VPN has to exist in the context of a connection.